test/extended/util/openshift/clusterversionoperator/adminack.go

```diff
@@ -25,6 +25,7 @@ import (
 type AdminAckTest struct {
 	Oc     *exutil.CLI
 	Config *restclient.Config
+	Poll   time.Duration
```

Comment:
I set it over in `test/e2e/upgrade/adminack/adminack.go`, because only the update suite needs the polling:

```console
$ git log -p origin/master...origin/pr/26649 | grep 'diff \|Poll' | grep -B1 Poll
    test/e2e/upgrade/adminack: Poll gates for duration of update
diff --git a/test/e2e/upgrade/adminack/adminack.go b/test/e2e/upgrade/adminack/adminack.go
+       adminAckTest := &exutil.AdminAckTest{Oc: t.oc, Config: t.config, Poll: 10 * time.Minute}
diff --git a/test/extended/util/adminack.go b/test/extended/util/adminack.go
+       Poll   time.Duration
+       if t.Poll == 0 {
+       if err := wait.PollImmediateUntilWithContext(ctx, t.Poll, func(ctx context.Context) (bool, error) {
```

Reply:
... which I'd accidentally dropped while rebasing :/.  Restored with 79d0aa9ec5 -> 4d1a25aee9.